src/cpp/rediscluster.cpp

```diff
@@ -126,6 +126,43 @@ CommandReply RedisCluster::run(AddressAnyCommand &cmd)
     return _run(cmd, _last_prefix);
 }
 
+// Run a non-keyed Command that addresses any db node on the server
```

Comment:
Should 'server' be 'cluster'?

Reply:
We use the terms interchangeably throughout this file